src/app/helptext/system/bootenv.ts

```diff
@@ -20,7 +20,11 @@ export const helptext_system_bootenv = {
   clone_source_tooltip: T("This is the boot environment to be cloned."),
 
   create_name_placeholder: T("Name"),
-  create_name_tooltip: T("Enter the name of the boot entry."),
+  create_name_tooltip: T(
+    "Enter the name of the boot entry.\
+ Names may include alphanumeric characters as well as\
```

Comment:
See #2896, but since these are all the same, this would be better as a shared string to not have repeated literal strings to get out of sync.

Reply:
That'd probably make sense.  The clone entry had a different first sentence so I don't know you want to handle that.  Making them all the same would be easier, I assume at some point in the past the tooltip dialog was different